lib/remote.js

```diff
@@ -1,34 +1,44 @@
 'use strict';
 
-const unzip = require('node-unzip-2');
+const path = require('path');
+const fs = require('fs-extra');
+const unzip = require('adm-zip');
 const config = require('./config');
 const axios = require('axios');
 
 // Downloads the zip file from github and extracts it to folder
-exports.download = (path) => {
-  const url = config.get().repository;
+exports.download = (loc, lang) => {
+  // If the lang is english then keep the url simple, otherwise add language.
+  const suffix = (lang === 'en' ? '' : '.' + lang);
+  const url = config.get().repositoryBase + suffix + '.zip';
+  const folderName = path.join(loc, 'pages' + suffix);
+  const REQUEST_TIMEOUT = 10000;
```

Comment:
Regarding this, we recently added support for the English archive in the same format `tldr-pages-en.zip` (after a request from tealdeer's maintainer). Can it be implemented here instead?

Reply:
sure should be easy.

Reply:
It should be noted that only the archive name should be generalized here, while still leaving in place that the english archive is extracted to the `pages` directory. Changing that behavior is a much larger one, which should be done in a separate PR that's focused purely on that, as will also have to take into account backwards compatibility.

Reply:
So, won't it be better to handle this entirely in the same (other) PR?

Reply:
I'd say so, but not sure how much @kbdharun wants to see the archive name changed here. The old zip will continue to work anyway for the time being that it's not exactly urgent to make that change either.